contracts/rollupReddit.sol

```diff
@@ -5,12 +5,15 @@ import {IReddit} from "./interfaces/IReddit.sol";
 import {ParamManager} from "./libs/ParamManager.sol";
 import {Types} from "./libs/Types.sol";
 import {NameRegistry as Registry} from "./NameRegistry.sol";
+import {RollupHelpers} from "./rollup.sol";
+import {RollupUtils} from "./libs/RollupUtils.sol";
 
-contract RollupReddit {
+contract RollupReddit is RollupHelpers {
```

Comment:
What's the reason we move the submitBatch out of rollup.sol to rollup reddit?

Reply:
Oh I thought that RollupReddit is the core rollup contract, is it not?

Reply:
I thought rollup.sol was what we get from hubble-core and we are going to use rollupReddit as the coordinator frontend contract.

Reply:
Yeah, RollupReddit is the core rollup contract. Ah ok I see what we are doing here.